src/sync/index.js

```diff
@@ -159,6 +168,11 @@ async function sync(db) {
         // must ensure updateCentralizedOraclesPassedResultSetEndBlock after updateOraclesPassedEndBlock
         await updateCentralizedOraclesPassedResultSetEndTime(currentBlockTime, db);
 
+        if (_.isNil(chainBlockNum)) {
+          logger.warn('chainBlockNum should not be null');
+        } else if (startBlock >= chainBlockNum) {
+          pubsub.publish('BatchSynced', { BatchSynced: { blockNum: currentBlockChainHeight } });
```

Comment:
Can we return a `SyncInfo` object here? Otherwise, the UI client has to do an extra call to get the `SyncInfo` for the new block. Seems redundant, the purpose of the subscription is to get new data instantly.